upgrade/Stage_0_Prerequisites.md

```diff
@@ -46,6 +46,16 @@ after a break, always be sure that a typescript is running before proceeding.
 
 1. (`ncn-m001#`) Install the latest `docs-csm` RPM.
 
+   - This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts.
+   CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
```

Comment:
Maybe it's just the diff.. but the spacing looks incorrect. This should be tabbed over to align with `This`. Please double-check this throughout the document.